history/warren-weaver.md

```diff
@@ -0,0 +1,11 @@
+---
+description: One of the pioneers of machine translation
+---
+# Warren Weaver
+Warren Weaver was an American scientist who designed several methods for developing Machine translation in 1949.
+
+The methods included the use of wartime cryptography techniques, Shannon’s information theory, 
+statistical analysis and juxtaposition of logic and universal features of language.
+
+With this method, research in Machine translation was ignited and the first conference was convened. 
```

Comment:
"With this method"...  doesn't make sense to me here.  And passive voice is usually a sign that we're being vague.

> His work sparked more research in machine translation and the first conference.